docs/diagrams/README.md

```diff
@@ -68,13 +69,17 @@ There are two levels of authentication:
     - We store our CA certificate (truststore.pem) in an S3 bucket in order for API Gateway to verify the identity of a consumer application.
     - We store our private key in a Kubernetes secret (api-gateway-certificate-authority) so that we can generate client certificates.
 2. For the consumer application, we create a client certificate (client.pem), which includes their public key. We also create their private key (client.key).
-3. We send both the client certificate (client.pem) and the private key (client.key) to the consumer application to be used for mutual TLS.
+3. We send both the client certificate (client.pem) and the private key (client.key) to the consumer application to be used for mutual TLS. 
+4. To inspect the certificate, retrieve and decode the public key, using this command to get it: `kubectl get secret mutual-tls-auth -n hmpps-integration-api-$environment -o json | jq -r '.data."truststore-public-key"' | base64 --decode`. Then decode the SSL certificate.
```

Comment:
Again, not sure this is the right place. Client certificates won't be in kubernete secret